.gitignore

```diff
@@ -103,6 +103,7 @@ tools/docfx/
 /src/Umbraco.Web.UI/appsettings-schema.json
 /src/Umbraco.Web.UI/appsettings-schema.*.json
 /src/Umbraco.Web.UI/umbraco-package-schema.json
+/src/Umbraco.Web.UI.Client/umbraco-package-schema.json
```

Comment:
This should be moved to the local `.gitignore` file inside the Client folder. Please also verify if the file `custom-elements.json` is properly ignored.